README.adoc

```diff
@@ -25,6 +25,42 @@ Site content for the Ansible Bootcamp Enablement.
     └── example_partial.adoc
 ----
 
+=== Using ETX-Ansible DevSpaces
+
+Once you have a lab environment set up, you can use DevSpaces to develop additional content. To do this you should first access your devspace environment by clicking the 9 square icon in the top right of the console and clicking, "Red Hat OpenShift DevSpaces".
```

Comment:
Update Dev Spaces to be two words "Dev Spaces" to align with the name of the product